docs/data/whats-new/WhatsNewLayout.js

```diff
@@ -0,0 +1,240 @@
+      },
+      {
+        title: 'ApiRef in community',
+        url: 'https://mui.com/blog/mui-x-v6/#apiref-moved-to-the-mit-community-version',
+      },
+      {
+        title: 'Cell selection',
+        url: 'https://mui.com/blog/mui-x-v6/#cell-selection',
+      },
+    ],
+  },
+  {
+    title: 'Date Pickers v5.0.0',
+    description:
+      'After some months of polishing in pre-releases, the Date Pickers finally get a stable.',
+    announcementDate: 'Monday, Sep 22, 2022',
+    url: 'https://mui.com/blog/date-pickers-stable-v5/',
+    highlightList: [
+      {
+        title: 'Better APIs',
+        url: 'https://mui.com/blog/date-pickers-stable-v5/#better-apis-and-improved-customization',
+      },
+      {
+        title: 'Easier customization',
+        url: 'https://mui.com/blog/date-pickers-stable-v5/#better-apis-and-improved-customization',
+      },
+      {
+        title: 'Integrated localization',
+        url: 'https://mui.com/blog/date-pickers-stable-v5/#integrated-localization',
+      },
+    ],
+  },
+  {
+    title: 'Data Grid v5.15',
+    description:
+      'This version brings an amazing set of new supported use cases with the Data Grid Premium.',
+    announcementDate: 'Monday, Aug 12, 2022',
+    url: 'https://mui.com/blog/aggregation-functions/',
+    highlightList: [
+      {
+        title: 'Aggregation in summary rows and row groups',
+        url: 'https://mui.com/blog/aggregation-functions/#wait-what-is-an-aggregation-function',
+      },
+      {
+        title: 'Row pinning',
+        url: 'https://mui.com/blog/aggregation-functions/#row-pinning',
+      },
+    ],
+  },
+  {
+    title: 'New Premium plan',
+    description:
+      'Premium plan announcement, including the most advanced features for data analysis and management.',
+    announcementDate: 'Thursday, May 12, 2022',
+    url: 'https://mui.com/blog/premium-plan-release/',
+    highlightList: [
+      { title: 'Row Grouping', url: '/x/react-data-grid/row-grouping/' },
+      { title: 'Excel export', url: '/x/react-data-grid/export/#exported-rows' },
+    ],
+  },
+  {
+    title: 'MUI X v5.0.0',
+    description: 'A new virtualization engine, and improvements in several APIs.',
+    announcementDate: 'Monday, Nov 22, 2021',
+    url: 'https://mui.com/blog/mui-x-v5/',
+    highlightList: [
+      {
+        title: 'New virtualization engine',
+        url: 'https://mui.com/blog/mui-x-v5/#a-new-virtualization-engine',
+      },
+      {
+        title: 'Improved state management',
+        url: 'https://mui.com/blog/mui-x-v5/#improved-state-management',
+      },
+      {
+        title: 'Simplified style customization',
+        url: 'https://mui.com/blog/mui-x-v5/#simplified-style-customization',
+      },
+    ],
+  },
+];
+
+function BlogCard(props) {
+  return (
+    <Card
+      sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        px: 2,
```

Comment:
A bit more width to read the text could be nice:

```suggestion
        px: 1,
```